packages/data/src/redux-store/test/index.js

```diff
@@ -281,7 +281,6 @@ describe( 'resolveSelect', () => {
 
 	it( 'returns only store native selectors and excludes all meta ones', () => {
 		expect( Object.keys( registry.resolveSelect( 'store' ) ) ).toEqual( [
-			'hasResolvingSelectors',
```

Comment:
why does this need to be removed?

Reply:
It was wrongly added in #50222 in the first place -`hasResolvingSelectors` is a meta selector and should not be stored at all by `resolveSelect`.

Reply:
FWIW it is being addressed separately in #52038.